crates/ruff_formatter/src/lib.rs

```diff
@@ -113,6 +114,12 @@ impl Default for IndentWidth {
     }
 }
 
+impl fmt::Display for IndentWidth {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        std::write!(f, "{}", self.0)
```

Comment:
For calls to `write!` that just print out the `Display` of a single argument like this, you can just do `Display::fmt(&self.0, f)` directly. (Technically `self.0.fmt(f)` can also work, but IIRC there are cases where that call is ambiguous with `Debug::fmt`. In which case, you'd get a compile error. Not sure if that's the case here though, so I tend to just write it out as `Display::fmt(...)`.)